specification/dimensions/pricingmodel.md

```diff
@@ -0,0 +1,39 @@
+# Pricing Model
+
+Pricing Model indicates the type of unit price applied to a charge at the time it was used or purchased. Pricing Model is commonly used to differentiate between charges at on-demand vs. reduced prices when investigating optimization opportunities, like for commitment-based discount coverage.
+
+The PricingModel column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the allowed values. PricingModel MUST be "Commitment-Based Discount" when CommitmentDiscountId is not null. PricingModel MUST be "Dynamic" when pricing is determined by the provider and may change over time, regardless of predetermined agreement pricing.
+
+## Column ID
+
+PricingModel
+
+## Display name
+
+Pricing Model
+
+## Description
+
+Type of unit price applied to a charge at the time it was used or purchased.
+
+## Content constraints
+
+|    Constraint   |      Value       |
+|:----------------|:-----------------|
+| Column required | True             |
+| Data type       | String           |
+| Allows nulls    | False             |
+| Value format    | list-of-values   |
+
+Allowed values:
+
+| Value                     | Description                                                                                                                                                                                                  |
+| :------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Fixed                     | Charges priced at a constant rate, regardless of quantity used or purchased.                                                                                                                                 |
+| On-Demand                 | Charges priced at the standard rate for the billing account. This includes any applicable negotiated discounts and volume/tiered pricing, but does not include dynamic or commitment-based discount pricing. |
```

Comment:
```suggestion
| Standard                | Charges priced at the standard rate for the billing account. This includes any applicable negotiated discounts and volume/tiered pricing, but does not include dynamic or commitment-based discount pricing. |
```

"On-Demand" here is a tricky term.  To my mind, the primary definition of that term is “I ran this resource whenever I wanted” instead of “I paid the non-discounted rate”.  (I can respect the perspective that “On Demand” might sometimes mean one or the other — but our own [current glossary draft](https://docs.google.com/document/d/1PvJrjXSeqtrbVAHW-aht1gzLM47EJnzX3jmjWQ8ZxkE/edit) makes no mention of rate when defining the term.)  A resource can be both run on demand _and_ covered by a CBD, and as such, these proposed groupings become potentially confusing for the practitioner.

In terms of FOCUS values in a Pricing Model attribute, how about sidestepping that semantic conflict and using a term like “**Standard**” (or “Full”) instead of “On Demand”?

Reply:
VS2 Squad considered this, and we have decided that On-Demand is a more generic, understood term than Standard, which would invite discussions as it is broader. We will leave it as On-Demand for now and move this to Maintainers for review. This conversation will not be closed though to invite further comments on this.

Reply:
To me, from a pricing / purchasing model perspective, on-demand means it was purchased without any prior commitment. It has nothing to do with if a negotiated rate was applied or not. The same row has metrics that have metrics that are with negotiated discounts and some without. I'm good with on-demand instead of standard/fixed IF we can make this nullable and not have everything default to on-demand which reduces the value of this column (tax, support, and to a lesser degree, purchases aren't on-demand IMO). If it has to be non-nullable, then it feels like we need another value that can cover some of the cases like tax, support etc.

Reply:
I agree with Udam's proposition to make Pricing Model nullable, because it's about price. For example a tax has arguably a price. Support has a price: maybe support could be charged based on actual hourly support, so on-demand, or on a fixed monthly amount?

> Pricing Model indicates the type of unit price applied to a charge at the time it was used or purchased.

Reply:
Ok, we have now added NULLABILTY (to be used when SkuPriceID is NULL and an OTHER option (for when a Pricing Model is present, but does not fit in to any of the other options (we can then expand on this in a future version once we capture more data). Please review and tell me if your comments above have been addressed or if more changes are needed...